### DISCUSSION TITLE
Recently played in the last N days

### DISCUSSION THREAD

**Discussion by Wibblyw** [Chimildic/goofy#257] [Помощь с алгоритмом] [answered]
Can anyone please suggest a suitable method for getting an array of the RecentlyPlayed tracks within the last N days?

**Comment by Chimildic** [comment, marked as answer]
[rangeDateRel](https://chimildic.github.io/goofy/#/reference/filter?id=rangedaterel) or [rangeDateAbs](https://chimildic.github.io/goofy/#/reference/filter?id=rangedateabs)
```js
let recentTracks = RecentTracks.get()
Filter.rangeDateRel(recentTracks, 7, 0)
```

**Reply by Wibblyw** [reply]
Thanks, will try that :-)
